index.js

```diff
@@ -1,18 +1,20 @@
 const gq = require('./services/goquest_service');
 var cron = require('node-cron');
 
-let cronStarted = false;
-if (!cronStarted) {            
+// let cronStarted = false;
+// if (!cronStarted) {
 
-    gq.openConnection();
 
-    // check games every minute
-    cron.schedule('0 */3 * * * *', () => {
+// check games every minute
+// cron.schedule('0 */3 * * * *', () => {
 
-        gq.openConnection();
-        
-    });
+// gq.openConnection();
 
-    cronStarted = true;
+// });
 
-}
\ No newline at end of file
+// cronStarted = true;
+
+// }
+exports.handler = (event, context, callback) => {
```

Comment:
make it compatible with the heroku index.

Maybe we just need to export the handler and disable the cron via env vars